src/localPRG.cpp

```diff
@@ -304,11 +280,38 @@ std::vector<Path> LocalPRG::shift(Path p) const {
             }
         }
         if (non_terminus) {
-            return_paths.push_back(k_paths[i]);
+            shifted_paths.push_back(p);
         }
     }
 
-    return return_paths;
+    return shifted_paths;
+}
+
+std::vector<Path> LocalPRG::get_extended_paths(const Path &path) const {
+    std::deque<Path> short_paths = {path};
+    std::vector<Path> k_paths = {};
```

Comment:
@rmcolq What is `k_paths`? Kmer paths?

Reply:
Thanks for the response. What is `k` please?

Reply:
`k` is an input parameter and determines the length of the (w,k)-minimizers (smallest kmer in window of w consecutive kmers). I usually use `k=15`, but for other kmer-based work values are often e.g. `21`, `31`.

Reply:
Would `min_kmer_length` be an accurate name?

Reply:
`k` is `kmer_length`. All kmers have to be exactly `k` bases long. Looking again, `k_paths` are the paths of length exactly `k` and `short_paths` are the paths of length less than `k` which I'm extending to paths of length `k`.